src/math/p5.Vector.js

```diff
@@ -1742,6 +1742,156 @@ p5.Vector = class {
     return this;
   }
 
+  /**
+ * Performs spherical linear interpolation with the other vector
+ * and returns the resulting vector.
+ * The result of slerping between 2D vectors is always a 2D vector.
```

Comment:
By only mentioning 2D vectors in the description, I worry people might incorrectly assume it's *only* for 2D. Maybe we can say "This works in both 3D and 2D" before this sentence?

Reply:
Spherical linear interpolation is usually used in 3D, so I didn't mention it, but I would like to describe it if necessary (I wanted to emphasize that the result is 2D even if each 2D vectors are pointered directions oppositely).